log/logger.go

```diff
@@ -13,7 +13,7 @@ import (
 type Logger interface {
 	With(ctxs ...Context) Logger
 	WithMap(mapCtx map[string]string) Logger
-	WithKeyValue(keyvals ...string) Logger
+	WithKeyValue(key, value string) Logger
```

Comment:
Nitpick:
Could we shorten this to `WithKV`?

Reply:
I would make it `With` and `WithContext` depending on the frequency of usage :)

From my recent experience I used `WithKeyValue` a lot and zero times `With`.

Reply:
@InfernoJJ what's your experience?

Reply:
Agreed, looking at our `identity` service, `WithKeyValue()` is called a lot more frequently than `With(ctx ...Context)`. 
Renaming:
- `WithKeyValue` -> `With`
- `With` -> `WithContexts` 
would be great.

Reply:
@InfernoJJ thoughts?

Reply:
@vxio let's keep it as it. we can rename it pretty quickly if all projects together. Right now it may be a little bit complicated.